ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -519,8 +519,8 @@ class MSFAvatarStateImpl: NSObject, ObservableObject, MSFAvatarState {
         if !isRingVisible {
             return avatarImageSize + (ringOuterGap * 2)
```

Comment:
Why isn't the avatar's size just image size when it doesn't have a ring? Also, I think it might just draw the avatars at this size, rather than the image size? If you draw a bunch of no ring avatars with 0 spacing, their edges touch, until the ring avatars where each one has an extra gap around it.

Reply:
It was to align with when the avatars with ring have an extra gap, but these changes make sense!

Reply:
Ah, makes sense